### ISSUE TITLE
Question: what kind of task/labels can this library supports?

### ISSUE THREAD

**Issue by biagio-lunit** [jamesdolezal/slideflow#210]
Is this framework specifically for image classification only, or can it also be used / adapted for cell detection and nuclei segmentation? If not, is there any plan to add these tasks? Thanks!

**Comment by jamesdolezal** [comment]
Cell detection and nuclei segmentation are under development! We are hoping to include these tasks in version 1.5, which we aim to release around February. In the meantime, Slideflow supports classification, regression, and time-to-event prediction.

**Comment by jamesdolezal** [comment]
Just to follow up - Version 1.5 has been released, and with this release Slideflow supports:

- Weakly-supervised tile-based models (CNNs and transformers) for classification, regression, and time-to-event
- Weakly-supervised multi-instance learning (CLAM) for classification
- Self-supervised learning (SimCLR)
- Generative adversarial networks (StyleGAN2 and StyleGAN3)
- Whole-slide cell segmentation (Cellpose)
- Custom training loops and architectures by providing a preprocessed Tensorflow or PyTorch dataloader

Thanks, and closing this issue for now.
